api/src/main/java/net/kyori/adventure/text/NBTComponent.java

```diff
@@ -26,9 +26,13 @@
 import org.checkerframework.checker.nullness.qual.NonNull;
 
 /**
- * An NBT component.
+ * A component that can display NBT fetched from different locations, optionally trying to interpret the NBT as JSON
```

Comment:
```
A {@link Component} which formats and displays an object that is defined via NBT.

@see ...
```

How the NBT component might go about that, i.e. interpreting JSON, is an implementation detail.

Reply:
In this case it is a relevant part of API -- because of the `interpret` flag. Any discussion of the JSON text format should also reference `GsonComponentSerializer` (though in this case it can't really link directly because of submodules, so you'd have to specify it in a `{@code ...}` block)